data/mods/My_Sweet_Cataclysm/sweet_mutation_category.json

```
@@ -4,8 +4,7 @@
     "id": "SUGAR",
     "name": "Sugar Kin",
     "threshold_mut": "SUGAR",
-    "mutagen_message": "How sweet!",
-    "iv_message": "You inject some more fizz into your syrup.",
+    "mutagen_message": "How sweet, this blissful feeling!",
```

```suggestion
    "mutagen_message": "How sweet!",
```
No need to edit the message, especially since you can only gain the mutation at char gen and not in game.